new/src/components/ScenarioOverview/ScenarioOverview.integration.test.tsx

```diff
@@ -0,0 +1,32 @@
+import { render, screen } from "@testing-library/react";
+import { ScenarioOverview } from "../Scenarios/ScenarioOverview";
+import userEvent from "@testing-library/user-event";
+import { MemoryRouter } from "react-router-dom";
+
+jest.mock("./DonutChart", () => ({
+    createReportCircle: () => null
+}));
+
+describe("<ScenarioOverview/>", () => {
+    const reportName = "My report";
+    const description = "My description";
+    const title = "My title";
+
+    it("should only show failed scenarios after clicking the link to filter for failed scenarios", () => {
```

Comment:
Since this test is green by mistake, we should skip it so we know it is a technical debt